Script/material.dat

```diff
@@ -2249,6 +2336,15 @@ flesh
     AttachedGod = SCABIES;
     BodyFlags = Base&~IS_WARM_BLOODED;
   }
+
+  Config ELF_FLESH;
+  {
+    NameStem = "elven flesh";
+    AttachedGod = MORTIFER;
+    Effect = EFFECT_CONFUSE; /* elven flesh is sweet and addictive */
```

Comment:
`EffectStrength` should be set to something between`10` and `30` to prevent the player from gaining permanent or near-permanent confusion from a single mistress corpse.